app/assets/stylesheets/modules/map/map.css.scss

```diff
@@ -306,6 +306,32 @@ p.map-popup-info {
   left: 50%;
 }
 
+.map-grid-search {
+  color: #fff;
+  z-index: 59;
+  padding: 10px 15px;
+  font-size: 1em;
+
+  input {
+    padding: 0;
+    width: 16em;
+    height: 22px;
+  }
+
+  label {
+    min-width: 0;
+  }
+}
+
+.landing-toolbar-content .error {
+  max-width: 590px;
```

Comment:
This is more like a hack to me - is there a place that defines the max width of the filter stack? @mreese84